packages/mathtype-ckeditor5/src/integration.js

```diff
@@ -254,11 +254,18 @@ export default class CKEditor5Integration extends IntegrationModel {
             } );
 
         } else {
-            returnObject.node = this.editorObject.editing.view.domConverter.viewToDom(
-                this.editorObject.editing.mapper.toViewElement(
-                    this.insertMathml( mathml )
-                ), windowTarget.document
-            );
+            try {
+                returnObject.node = this.editorObject.editing.view.domConverter.viewToDom(
+                    this.editorObject.editing.mapper.toViewElement(
+                        this.insertMathml( mathml )
+                    ), windowTarget.document
+                );
+            }  catch (e) {
+                const x = e.toString();
+                if((x).includes("CKEditorError: Cannot read property 'parent' of undefined")) {
```

Comment:
We are trying to follow CKEditor 5 conventions for this plugin. In particular, try to leave a single whitespace after `(` and before `)`, e.g. `function( argument )`. Same for `[`, `]`, `{`, and `}`. See: https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/code-style.html

We are not too strict with this, though, but it's good to have in mind for consistency.